components/settings/FontSizeCounterInput.tsx

```diff
@@ -15,8 +15,8 @@ export default function FontSizeCounterInput() {
 	const increaseFontsize = () => {
 		if (fontSize < 20) {
 			setFontSize((prev) => {
-				const newFontSize = prev + 1;
-				localStorage.setItem("fontSize", newFontSize);
+				const newFontSize:number = prev + 1;
```

Comment:
the type number is inferred by `prev`. there is no need to declare this

Reply:
noted